float64 gripper
float64 arm_1
float64 arm_2
float64 arm_3
float64 arm_4
float64 arm_5
float64 arm_6
